modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/transformers/ActionsComponentPropsTransformer.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {render} from 'frontend-js-react-web';
+import React from 'react';
+
+import SidebarPanel from '../SidebarPanel';
+
+export default function propsTransformer({
+	items,
+	sidebarContainerSelector,
+	...otherProps
+}) {
+	const sidebarRefPanel = React.createRef();
+
+	const actions = {
+		showInfo() {
+			showSidebar();
+		},
+		showMetrics() {
+			showSidebar();
+		},
+	};
+
+	const hideSidebar = () => {
+		sidebarRefPanel.current.close();
+	};
```

Comment:
This seems like an odd pattern to use `useImperativeHandle` but then pass this back into the component for its `onClose`. I would expect that onClose would be handled internally, unless we intend to add more logic in here.

Reply:
Yep, we don't need this. We only would need this in case we want to add more logic onClose, but it's not the case.